src/pvnet_app/data/nwp.py

```diff
@@ -148,10 +144,9 @@ def preprocess_nwp_data(use_ukv: bool | None = True, use_ecmwf: bool | None = Tr
 
         # UKV data must be float16 to allow overflow to inf like in training
         fix_ukv_data()
-    else:
```

Comment:
could you include a logging, saying `nwp_ukv_path` doesnt exists, just so its clear why something didnt happen